src/components/ProductsList/ProductsList.tsx

```diff
@@ -107,12 +170,8 @@ function ProductsList() {
             <section className={styles.productsList}>
                 <div className="container">
                     <div className={styles.wrapper}>
+                        {isMobile && isFetching && data && data.length === 0 && <p>Loading Products...</p>}
```

Comment:
move this condition `isMobile && isFetching && data && data.length === 0` into separate variable